code/style.css

```diff
@@ -0,0 +1,314 @@
+/* ---------------------- GENERAL STYLING ---------------------- */ 
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&display=swap');
+
+* {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    -webkit-font-smoothing: antialiased;
+    text-rendering: optimizeLegibility;
+}
+
+html {
+    font-family: 'Roboto Mono', monospace;
+    background: #000;
+    color: #fff;
+}
+
+body {
+    padding: 0;
+    width: 70vw;
+    margin: auto;
+    text-shadow: 1px 1px #004D16;
+}
+
+h1 {
+    font-size: 40px;
+}
+
+h2 {
+    margin: 0 auto 30px auto;
+    padding: 0;
+    text-align: center;
+}
+
+p {
+    margin: 0;
+    padding: 0;
+}
+
+/* ---------------------- HEADER ----------------------*/ 
+header {
+    margin: 60px 20px 30px 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    transition: margin 0.5s linear;
+}
+
+header img {
+    width: 5vw;
+    text-align: center;
+}
+
+/* ---------------------- MAIN ----------------------*/ 
+main {
+    text-align: left;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-bottom: 100px;
+}
+
+.marginTop {
+    margin-top: 150px;
+}
+
+/*  PROGRESS-BAR  */ 
+.progress-bar-container-outer {
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 100vw;
+    height: 95px;
+    background: #000;
+    border-bottom: 2px solid #01DC3E;
+    transition: visibility 0s, opacity 0.5s linear;
+}
+
+.progress-bar-container-inner {
+    width: 70vw;
+    margin: auto;
+    padding: 20px;
+    font-size: 12px;
+}
+
+.progress-bar-container-inner p {
+    margin: 5px 0;
+}
+
+.progress-bar-container-inner-content {
+    display: flex;
+    justify-content: space-between;
+}
+
+.progress-bar {
+    width: 100%;
+    display: flex;
+}
+
+.progress-bar div {
+    height: 1vh;
+    width: 50%;
+    background: #333;
+}
+
+/*  CHAT-BOXES  */ 
+.chat-box {
+    display: flex;
+    align-items: center;
+    margin: 10px 0;
+    transition: opacity 0.5s linear;
+}
+
+.pizzeria p {
+    color: #fff;
+    background: #01DC3E;
+    width: 30vw;
+    padding: 20px;
+    border-radius: 10px;
+    margin-left: 20px;
+}
+
+.user {
+    align-self: flex-end;
+}
+
+.user-inner {
+    padding: 15px 20px 15px 20px;
+    border-radius: 10px;
+    color: #fff;
+    border: 2px solid #999;
+    margin-right: 20px;
+}
+
+/*  PORTRAITS  */ 
+.chat-portrait {
+    width: 80px;
+    height: 80px;
+    border: 2px solid #01DC3E;
+    border-radius: 50%;
+    background-image: url(./images/pizza-slice2.png);
```

Comment:
Some browsers will not format that path because of lack of quotes. It's far more safe to define `background-image` property like this :

`background-image: url("./images/pizza-slice2.png");`